packages/deployments/contracts/contracts/messaging/interfaces/ambs/Multichain.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity 0.8.15;
+
+/**
+ * @dev interface to interact with multicall (prev anyswap) anycall proxy
+ *     see https://github.com/anyswap/multichain-smart-contracts/blob/main/contracts/anycall/AnyswapV6CallProxy.sol
+ */
+interface Multichain {
```

Comment:
im a bit unclear on how this Multichain/Multicall stuff factors into the current design... is this the thing that xapp devs would be implementing?

Reply:
no this is the AMB we are using to connect to binance